### ISSUE TITLE
Disable but: call to eval() blocked by CSP

### ISSUE THREAD

**Issue by wellington1993** [emberjs/ember-inspector#659]
Hi,

Sometimes I got this error message from console on some non ember pages:
```
console.error: ember-inspector: 
Object
    - _errorType = Error
    - message = call to eval() blocked by CSP
    - fileName = 
    - lineNumber = 0
    - stack = 
    - name = Error

```
The strange is the message still appears after disable the extension.
I'm have the Firefox Nightly as Browser.

Sorry if duplicated or irrelevant.

Thanks.

**Comment by sandstrom** [comment]
I think this is a duplicate of https://github.com/emberjs/ember-inspector/issues/471 and can be closed.
